graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -358,38 +256,42 @@ def _get_selectivity_fraction_of_interval(interval, quantiles):
     of the whole domain.
 
     Args:
-        interval: IntegerInterval defining the range of values
-        quantiles: a sorted list of N values separating the values of the field into N-1
-                   groups of almost equal size. The first element of the list is the
+        interval: Interval[T] defining the range of values
+        quantiles: a sorted list of N values of type T separating the values of the field
+                   into N-1 groups of almost equal size. The first element of the list is the
                    smallest known value, and the last element is the largest known value.
                    The i-th element is a value greater than or equal to i/N of all present values.
                    N has to be at least 2.
 
     Returns:
         float, the fraction of the values contained in the interval.
     """
+    if interval.is_empty():
+        return 0
```

Comment:
It's nice to reinforce the float type of the result for the sake of the reader
```suggestion
        return 0.0
```